src/App.js

```diff
@@ -1,23 +1,50 @@
-import logo from './logo.svg';
 import './App.css';
+import { useEffect, useState } from 'react';
+import { imageDesc, imageAsc } from './services/images';
+import ImageCard from './components/ImageCard';
+import Filter from './components/Filter';
 
 function App() {
+  const [pictures, setPictures] = useState([]);
+  const [order, setOrder] = useState('asc');
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      if (order === 'desc') {
+        const data = await imageDesc();
+        setPictures(data);
+        setLoading(false);
+      } else {
+        const data = await imageAsc();
+        setPictures(data);
+        setLoading(false);
+      }
+    };
```

Comment:
Cool solution. The API is a little slow, so a separate fetch for desc isn't ideal in practicality. However, as a this project is a testing exercise, no problem.